scripts/maintenance_effectiveness_indicator.py

```diff
@@ -54,7 +54,7 @@ def main() -> None:
     )
 
     for factor, value in result.items():
-        logger.info(f"{factor}: {round(value, 2) if value != math.inf else value}")
+        logger.info("%s : %d", factor, round(value, 2) if value != math.inf else value)
```

Comment:
The whitespace was not present in the original login statement:
```suggestion
        logger.info("%s: %d", factor, round(value, 2) if value != math.inf else value)
```